reana_workflow_controller/templates/dask_cluster.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: kubernetes.dask.org/v1
+kind: DaskCluster
+spec:
+  worker:
+    replicas: 0
+    spec:
+      containers:
+        - name: worker
+          imagePullPolicy: "IfNotPresent"
+          command: ["/bin/sh", "-c"]
+          args:
+            - exec dask-worker --name $(DASK_WORKER_NAME) --dashboard --dashboard-address 8788
+          ports:
+            - name: http-dashboard
+              containerPort: 8788
+              protocol: TCP
+  scheduler:
+    spec:
+      containers:
+        - name: scheduler
+          imagePullPolicy: "IfNotPresent"
+          args:
+            - dask-scheduler
+          ports:
+            - name: tcp-comm
+              containerPort: 8786
+              protocol: TCP
+            - name: http-dashboard
+              containerPort: 8787
+              protocol: TCP
+          readinessProbe:
+            httpGet:
+              port: http-dashboard
+              path: /health
+            initialDelaySeconds: 5
```

Comment:
It might be good to make these hard-coded options (such as the liveness probe seconds etc) configurable via environment variables. This could come handy for some deployment scenarios. We could also do this later after the initial basic version is merged.

(Alternatively, we could think of customising the whole template, but that would surely go beyond the scope of this initial PR.)

Reply:
I guess you did not address the comment on hard-coded values. Do we keep this or make the liveness probe etc configurable?

```
            initialDelaySeconds: 5
            periodSeconds: 10
```

(We can keep hard-coded if not terribly important, especially if we use some "safer" values for various deployment scenarios, such as slow cluster vs fast cluster, etc.)